charts/seed-bootstrap/charts/fluent-bit/templates/fluent-bit-configmap.yaml

```diff
@@ -4,7 +4,7 @@ metadata:
   name: fluent-bit-config
   namespace: {{ .Release.Namespace }}
   labels:
-{{ toYaml .Values.fluentbit.labels | indent 4 }}
+{{ toYaml .Values.labels | indent 4 }}
```

Comment:
`pkg/operation/common/utils.DeleteLoggingStack` is still using `garden.sapcloud.io/role` for deletion. Disabling the features gate does not delete the logging workload.

Reply:
nice catch! gonna fix this